gamemodes/terrortown/gamemode/client/vgui/cl_sb_row.lua

```diff
@@ -433,13 +432,23 @@ function PANEL:UpdatePlayerData()
 	self.tag:SetText(ptag and GetTranslation(ptag.txt) or "")
 	self.tag:SetTextColor(ptag and ptag.color or COLOR_WHITE)
 
-	self.sresult:SetVisible(ply.bodySearchResult and ply.bodySearchResult.base.isPublicPolicingSearch)
-
-	-- more blue if a detective searched them
-	if ply.bodySearchResult and (LocalPlayer():GetSubRoleData().isPolicingRole or not ply.bodySearchResult.show) then
-		self.sresult:SetImageColor(Color(200, 200, 255))
+	local show_sresult = true
+	-- we have body search results
+	if ply.bodySearchResult and ply.bodySearchResult.show_sb then
+		self.sresult:SetImage("icon16/information.png")
+		self.sresult:SetTooltip(GetTranslation("cl_sb_row_sresult_direct_conf"))
+	elseif ply:OnceFound() and not ply:RoleKnown() then
+		self.sresult:SetImage("icon16/comments.png")
+		self.sresult:SetTooltip(GetTranslation("cl_sb_row_sresult_in_conf"))
```

Comment:
Remove this indirect confirmation case and i'm fine with the PR.

Like i said in the thread on the language file i think this case doesn't add any more info for the player and only adds visual clutter.
Without this case the other two cases signal the Player "Click me! There is more information here!" which would be a great addition and would save players from clicking through every single scoreboard item to see if there is any search result.